src/components/DeckMechanisms/index.js

```diff
@@ -88,12 +92,24 @@ export default class DeckMechanisms extends React.Component {
     // If it’s not a discard move and the card costs more mana than the current
     // round, skip play.
     if (options.discard || canAfford) {
+      const harvestersActions = {
+        ...this.props.HoS,
+      }
```

Comment:
I don’t think you need to clone it. We’re not going to mutate it in the function. You can pass `this.props.HoS` directly.